packages/loader/src/gltf/extensions/GLTFExtensionParser.ts

```diff
@@ -26,7 +26,7 @@ export abstract class GLTFExtensionParser {
     extensionSchema: GLTFExtensionSchema,
     extensionOwnerSchema: GLTFExtensionOwnerSchema,
     ...extra
-  ): EngineObject | Promise<EngineObject> {
+  ): EngineObject | Uint8Array | Promise<EngineObject | Uint8Array> {
```

Comment:
EngineObject  | Promise<EngineObject | Uint8Array> is OK

Reply:
done